src/constants/index.ts

```diff
@@ -232,8 +250,12 @@ export const TBTC_META_SWAP_DEPOSIT_ADDRESSES = buildAddresses({
 })
 
 export const TBTC_META_SWAP_V2_DEPOSIT_ADDRESSES = buildAddresses({
-  [ChainId.MAINNET]: "0x05383312655856E25b851c15fA856dB7e270F0cF",
-  [ChainId.HARDHAT]: "0xB06Ce7334A26e90077F0182F07aCF650Bc978936",
+  [ChainId.MAINNET]: "0xee1ec4e1C6e39C31dAaf3db2A62A397bdf3fe2f1",
```

Comment:
I believe this is wrong + shouldn't be changed
https://github.com/saddle-finance/saddle-contract/blob/master/deployments/mainnet/SaddleTBTCMetaPoolUpdatedDeposit.json

Reply:
Yeah this should be left untouched